source/screens/App/components/AppEntry.js

```diff
@@ -1,30 +1,33 @@
 import React from 'react'
-import {
+import { 
   BrowserRouter as Router,
   Switch,
-  Route,
+  Route
 } from 'react-router-dom'
+
 import Logo from './Logo'
+import { SideMenuBar } from '../components'
+import { Home } from '../screens/Home'
 import { Jobs } from '../screens/Jobs'
 import { Leads } from '../screens/Leads'
 import { Materials } from '../screens/Materials'
 import { Quotes } from '../screens/Quotes'
-import { UserProfile } from '../screens/UserProfile'
 import { Vendors } from '../screens/Vendors'
```

Comment:
Was this deleted because you don't have it?

Reply:
Yeah I think so. I think I didn't have the latest.